### PR TITLE
cornmeal is actually obtainable now

### PR DESCRIPTION
**Pull request by Stealthbomber16**
## About the PR
You can actually grind corn into cornmeal now.

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

:cl:
- fix: You can actually grind corn into cornmeal now.

### CODE REVIEW

File: Resources/Prototypes/Entities/Objects/Consumable/Food/produce.yml [hunk 1]
```diff
@@ -814,6 +814,8 @@
     sprite: Objects/Specific/Hydroponics/corn.rsi
   - type: Produce
     seedId: corn
+  - type: Extractable
+    grindableSolutionName: food
```
Comment (ShadowCommander) [inline Resources/Prototypes/Entities/Objects/Consumable/Food/produce.yml:818]: Move this down a line. The test failure is that there's two ExtractibleComponents on this entity.

**Review by ShadowCommander — CHANGES_REQUESTED** [review]

File: Resources/Prototypes/Entities/Objects/Consumable/Food/produce.yml (hunk 1 above)
Comment (Stealthbomber16) [reply Resources/Prototypes/Entities/Objects/Consumable/Food/produce.yml:818]: Got it. Move the grindable down under the second type: extractable, and delete the first one? That will work?

**Comment by Stealthbomber16** [comment]
well shit

**Comment by Stealthbomber16** [comment]
i think that test fail is an error because i have no other explanation
its exactly the same formatting that tomatoes have at line 584

File: Resources/Prototypes/Entities/Objects/Consumable/Food/produce.yml [hunk 2]
```diff
@@ -815,6 +815,7 @@
   - type: Produce
     seedId: corn
   - type: Extractable
+      grindableSolutionName: food
```
Comment (ShadowCommander) [inline Resources/Prototypes/Entities/Objects/Consumable/Food/produce.yml:818]:
```suggestion
    grindableSolutionName: food
```

**Review by ShadowCommander — CHANGES_REQUESTED** [review]

**Comment by potato1234x** [comment]
wait what? i swear when i added cornmeal i made it obtainable...

**Comment by potato1234x** [comment]
yeah, does this not work anymore?
image

**Comment by Stealthbomber16** [comment]
grinding corn currently gives nutriment and vitamin
by adding the one line YML change i added, you should actually get the cornmeal

**Comment by potato1234x** [comment]
weird

**Review by ShadowCommander — APPROVED** [review]

**Comment by ShadowCommander** [comment]
Test it and then it should be good to merge.

**Comment by Stealthbomber16** [comment]
Couldn't get a dev environment to test this and now it's implemented and it still doesn't fucking work. I'm malding.

**Pull request merged**